### ISSUE TITLE
dropdown misspelled in check on ApiConnection [was exportRecords crash]

### ISSUE THREAD

**Issue by spgarbet** [vubiostat/redcapAPI#130]
A user is reporting the following when using exportRecords

```
Error in rccola::drinkREDCap(variables = clinref_token, FUN = read_data) : 

  1 assertions failed:

 * The field choice string does not appear to be formatted for choices.

Calls: <Anonymous> ... fieldChoiceMapping -> fieldChoiceMapping.character -> <Anonymous>
```

I've asked for their read_data code to narrow this down.

**Comment by spgarbet** [comment]
```
read_data <- function (
        key,
        ...) {
      redcapAPI::exportRecords(
        rcon = redcapConnection(url = rcon.api, token = key),
        labels = FALSE,
        factors = FALSE,
        records = records.dat,
        fields = vars.dat)
  }
```

**Comment by spgarbet** [comment]
I pushed the fix @nutterb recommend. I don't see how this fixes the issue. I think there's a failure in the REGEX for choices. I think the REDCap meta data has something that REGEX_MULT_CHOICE doesn't recognize. If that doesn't fix it, I will get a copy of the meta data to scan.

**Comment by nutterb** [comment]
Oh, you may be right. I was getting stuck on a different error message coming from the `redcapApiConnection` method. 

This would help us identify which field is kicking up the problem. 
```
MetaData <- rcon$metadata()
MetaData <- MetaData[MetaData$field_type %in% c("dropdown", "radio", "checkbox"), ]

for (i in seq_len(nrow(MetaData))){
  print(MetaData$field_name[i])
  fieldChoiceMapping(MetaData$select_choices_or_calculations[i], 
                     MetaData$field_name[i])
}
```

**Comment by spgarbet** [comment]
Looking further into it, that error message was changed on Mar 29th. We pushed a bug fix on April 11th that addressed this. This is something we already patched with #61

**Comment by spgarbet** [comment]
I changed the name to reflect the fix that was made.

**Comment by spgarbet** [comment]
I'm going to close this one since it was fixed on #61. They need to update and try again.
